File: src/rails/common/parser/XMLTags.java
package rails.common.parser;

public final class XMLTags {
    /* TAGS */
	public static final String GAME_TAG = "Game";
	public static final String DESCR_TAG = "Description";
	public static final String CREDITS_TAG = "Credits";
	public static final String PLAYERS_TAG = "Players";
	public static final String OPTION_TAG = "GameOption";
	public static final String GAMES_LIST_TAG = "GamesList";
    public static final String NOTE_TAG = "Note";
        
	/* ATTRIBUTES */
	public static final String NAME_ATTR = "name";
	public static final String TEXT_ATTR = "text";
	public static final String MIN_ATTR = "minimum";
	public static final String MAX_ATTR = "maximum";
	public static final String PARM_ATTR = "parm";
	public static final String TYPE_ATTR = "type";
	public static final String DEFAULT_ATTR = "default";
	public static final String VALUES_ATTR = "values";
	public static final String CLASS_ATTR = "class";
    public static final String FILE_ATTR = "file";
    
	public static final String VALUES_DELIM = ",";
	
    /* Used by ComponentManager. */
    public static final String COMPONENT_MANAGER_ELEMENT_ID = "ComponentManager";
    public static final String COMPONENT_ELEMENT_ID = "Component";
}

File: dotnet/RailsLib.Net/Net/Common/Parser/XmlTags.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace GameLib.Net.Common.Parser
{
    public sealed class XmlTags
    {
        /* TAGS */
        public const string GAME_TAG = "Game";
        public const string DESCR_TAG = "Description";
        public const string CREDITS_TAG = "Credits";
        public const string PLAYERS_TAG = "Players";
        public const string OPTION_TAG = "GameOption";
        public const string GAMES_LIST_TAG = "GamesList";
        public const string NOTE_TAG = "Note";

        /* ATTRIBUTES */
        public const string NAME_ATTR = "name";
        public const string TEXT_ATTR = "text";
        public const string MIN_ATTR = "minimum";
        public const string MAX_ATTR = "maximum";
        public const string PARM_ATTR = "parm";
        public const string TYPE_ATTR = "type";
        public const string DEFAULT_ATTR = "default";
        public const string VALUES_ATTR = "values";
        public const string CLASS_ATTR = "class";
        public const string FILE_ATTR = "file";

        public const char VALUES_DELIM = ',';

        /* Used by ComponentManager. */
        public const string COMPONENT_MANAGER_ELEMENT_ID = "ComponentManager";
        public const string COMPONENT_ELEMENT_ID = "Component";
    }
}
